src/test/java/ev3dev/utils/DataChannelRereaderTest.java

```diff
@@ -0,0 +1,83 @@
+package ev3dev.utils;
+
+import lombok.SneakyThrows;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Some tests of DataChannelRereader.
+ *
+ * @author David Walend
+ */
+public class DataChannelRereaderTest {
+
+    static Path tempDirectory;
```

Comment:
Is `static` safe here? I don't know if JUnit supports running tests in one class in parallel, but if it does, this might produce unexpected behaviour.

Reply:
Good call. I think there's a `@BeforeAll` `@AfterAll` pairing that will only be called once.